FreeAPS/Sources/Shortcuts/TempPresets/tempPresetEntity.swift

```diff
@@ -29,3 +29,21 @@ import Swinject
         return tp
     }
 }
+
+@available(iOS 16.0, *) struct tempPresetsQuery: EntityQuery {
```

Comment:
Please capitalize this `struct` name and the file name to conform to how other `struct`s in the project are named to far, i.e. 
-`tempPresetEntity.swift` -> `TempPresetEntity.swift`
- `tempPresetsQuery`-> `TempPresetsQuery`

Reply:
@dnzxy Fixed

Reply:
Looking at the file tree and your recent commits this actually has not been resolved. The file name is still lowercase.

Reply:
@dnzxy Xcode was misbehaving, fixed now